### ISSUE TITLE
Support for Angular 6

### ISSUE THREAD

**Issue by digoburigo** [krakenjs/zoid#184]
I'm not finding the solution to implement this library, i want to make a button that call an iframe that loads a js file that enable paymentes with credit card, etc (like paypal). Anyone managed to do it something similiar?

Thanks in advance.

**Comment by bluepnume** [comment]
Does this example work? https://github.com/krakenjs/zoid/blob/master/demo/frameworks/angular2/index.htm

**Comment by jsebrech** [comment]
The angular 2 approach didn't work for us in angular 6+. Instead we created an [angular wrapper component](https://github.com/digipolisantwerp/embeddable-widgets_component_angular). We've also wrapped zoid into our own library so the code can be a bit hard to follow, but basically if you check how it is used in the readme and look at the code in [widget.component.ts](https://github.com/digipolisantwerp/embeddable-widgets_component_angular/blob/master/src/widget/widget.component.ts) it should be possible to figure it out.

**Comment by Andrea1710** [comment]
Hello! Is there any example with latest Angular to work with Zoid?

**Comment by mnicpt** [comment]
@Andrea1710 We are close to publishing for Angular 2 but not Angular 6.  I will add this to our documentation backlog and will provide a working sample for you by end of day tomorrow.  Thank you.

**Comment by mnicpt** [comment]
@Andrea1710 Angular 6 is currently not supported and will add this to our current backlog.

**Comment by mstuart** [comment]
Fixed w/ https://github.com/krakenjs/zoid/pull/365
